code/modules/spells/spell_types/pointed/duffelbag_curse.dm

```diff
@@ -0,0 +1,56 @@
+/obj/effect/proc_holder/spell/pointed/duffelbagcurse
+	name = "Duffel Bag Curse"
+	desc = "A spell that summons a duffel bag demon on the target, slowing them down and slowly eating them."
+	school = "transmutation"
+	charge_type = "recharge"
+	charge_max	= 150
+	charge_counter = 0
+	clothes_req = FALSE
+	stat_allowed = FALSE
+	invocation = "BA'R A'RP!"
+	invocation_type = INVOCATION_SHOUT
+	range = 7
+	cooldown_min = 50
```

Comment:
This is once every 5 seconds at best right? that's quite strong, but it is wiz I suppose